Description: Fix a unicode string
 Everything uses six, but there's still a problem with a string that hasn't
 been converted to using six.u(), so this patch does that.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-02-11

--- python-wsme-0.6.orig/wsmeext/sphinxext.py
+++ python-wsme-0.6/wsmeext/sphinxext.py
@@ -41,7 +41,7 @@ def make_sample_object(datatype):
     if datatype is wsme.types.bytes:
         return six.b('samplestring')
     if datatype is wsme.types.text:
-        return u'sample unicode'
+        return six.u('sample unicode')
     if datatype is int:
         return 5
     sample_obj = getattr(datatype, 'sample', datatype)()
